pages/common/valgrind.md

```diff
@@ -0,0 +1,20 @@
+# valgrind
+
+> Wrapper for a set of expert tools for profiling, optimizing and debugging programs.
+> Commonly used tools include `memcheck`, `cachegrind`, `callgrind`, `massif`, `helgrind`, and `drd`.
+
+- Use the (default) Memcheck tool to show a diagnostic of memory usage by `program`:
+
+`valgrind {{program}}`
+
+- Use Memcheck to report all possible memory leaks of `program` in full detail:
+
+`valgrind --leak-check=full --show-leak-kinds=all {{program}}`
+
+- Use the Cachegrind tool to profile and log CPU cache operations of `program`:
+
+`valgrind --tool=cachegrind {{program}}`
```

Comment:
Do each of the tools have aliases you can use instead?

This comment isn't a request to change - it's more of a question from someone who's curious :smiley_cat: